tests/auth-rust-lock/rust-toolchain

```diff
@@ -1 +1 @@
-1.69.0
+1.71.1
```

Comment:
upgrade all toolchain to 1.75.0.

https://github.com/nervosnetwork/ckb/blob/develop/rust-toolchain.toml